lib/gates_cirq.h

```diff
@@ -285,8 +315,11 @@ struct CXPowGate {
   }
 };
 
-// The (exponent=phi/pi, global_shift=-0.5) instance of XPowGate.
-// This is a function in Cirq.
+/**
+ * The `(exponent = phi/pi, global_shift = -0.5)` instance of XPowGate.
+ * This is equivalent to an X gate with a global phase.
```

Comment:
I think this is not equivalent to an X gate with a global phase, similarly for ry and rz. It is basically XPowGate with a fixed global shift. The  matrix is e^{-i X rads / 2}. In a sense, this is also a generalization of the X gate. This is the same as GateRX in gates_qsim.h.

Reply:
Done (also ry and rz).